rotkehlchen/chain/ethereum/utils.py

```diff
@@ -166,3 +174,38 @@ def should_update_protocol_cache(cache_key: GeneralCacheType, *args: str) -> boo
             key_parts=[cache_key, *args],
         )
     return ts_now() - last_update_ts >= ETH_PROTOCOLS_CACHE_REFRESH
+
+
+def try_download_ens_avatar(
+        eth_inquirer: 'EthereumInquirer',
+        avatars_dir: Path,
+        ens_name: str,
+) -> None:
+    """
+    Handles ens avatar downloading.
+    1. Checks whether given ens name has avatar set
+    2. If it does -- downloads the avatar and saves it in `avatars_dir`
+    3. Updates last avatar check timestamp for the given ens name
+
+    May raise:
+    - RemoteError if failed to query chain
+    - RequestsError if got avatar url, but failed to download it
+    """
+    dbens = DBEns(eth_inquirer.database)
+    avatar_url = eth_inquirer.ens_avatar_lookup(ens_name)
+    dbens.set_last_avatar_update(ens_name=ens_name, timestamp=ts_now())
+    if avatar_url is None:
+        return  # Avatar is not set
+
+    try:
+        avatar = requests.get(avatar_url, timeout=10).content
+    except RequestException as e:
+        log.info(
```

Comment:
Wtf? Why info!??
```suggestion
        log.error(
```

Reply:
that's not what info is for though. Info is really for stuff that should appear only in normal application run. Like ... "starting api", "running in Windows", etc.

If you put log level INFO, you should have very very limited logs. This is not an info log, this is an error log. An incorrect avatar link set by the user is also an error.